test/e2e/helpers.go

```diff
@@ -292,7 +310,7 @@ func execOnHost(controlPlaneEndpoint, hostname string, f io.StringWriter, comman
 	if err != nil {
 		return err
 	}
-	port := "22" // Need to use port 50001 for VMSS when MachinePools are supported here.
+	port := e2eConfig.GetVariable("VM_SSH_PORT") // Or VMMS_SSH_PORT when MachinePools are supported
```

Comment:
Maybe I should use a constant here instead of the string itself--I see we do that elsewhere with e2eConfig items.